Apps/Laboratory/src/Models/PublicCovidTestRecord.cs

```diff
@@ -116,6 +116,12 @@ public PublicCovidTestRecord()
         [JsonPropertyName("resultLink")]
         public string ResultLink { get; set; } = string.Empty;
 
+        /// <summary>
+        /// Gets or sets the state of the response.
+        /// </summary>
+        [JsonPropertyName("statusIndicator")]
+        public string StatusIndicator { get; set; } = string.Empty;
+
```

Comment:
We don't need to add the StatusIndicator property.

Reply:
@MikeLyttle  The bug fix is also for missing data. we need to add CollectionDate, ResultDatetime and Statusindicator